unetsocket/c/samples/rs232_wakeup.c

```diff
@@ -0,0 +1,47 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Script to wakeup a modem over RS232.
+//
+// In terminal window (an example):
+//
+// $ make samples
+// $ ./rs232_wakeup <dev_name>
+//
+// NOTE: Validity of the <dev_name> is not checked in this example code.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <math.h>
+#include <inttypes.h>
+#include "../unet.h"
+#include "../unet_ext.h"
+
+static int error(const char *msg)
+{
+  printf("\n*** ERROR: %s\n\n", msg);
+  return -1;
+}
+
+int main(int argc, char *argv[])
+{
+  int ret = 0;
+  if (argc < 2)
+  {
+    error("Usage : rs232_wakeup <dev_name> \n"
+          "dev_name: Device name on which serial port exists. \n"
+          "A usage example: \n"
+          "rs232_wakeup /dev/tty.usbserial \n");
+    return -1;
+  }
+  ret = unetsocket_rs232_wakeup(argv[1], 115200, "N81");
```

Comment:
Do we want to try multiple times to wake it up before reporting wake up failed?

Reply:
Fixed in fe57ea5